### PR TITLE
WIP: Add redirect

### PR DESCRIPTION
**Pull request by Naomi-Dennis**
* Add headers to `HttpResponse` to process `SimpleApplication` redirect

⚠️ Depends on #10

### CODE REVIEW

File: src/main/java/http/HttpResponse.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package http;
+
+import java.util.ArrayList;
+
+public class HttpResponse{
+    public String status = "404";
+    public String body = "";
```
Comment (damonkelley) [inline src/main/java/http/HttpResponse.java]: Still not sure about these defaults. Let's keep on eye on this going forward.

File: src/main/java/clientApplication/SimpleApplication.java [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+public class SimpleApplication implements Application {
     public HttpResponse start(HttpRequest clientRequest){
         HttpResponse response = new HttpResponse();
 
         if(clientRequest.getResource().matches(".*simple_get|.*echo_body")){
             response.status = "200";
         }
+        else if(clientRequest.getResource().matches(".*redirect")){
+            response.status = "301";
+            response.headers.add("Location: http://127.0.0.1:5000/simple_get");
```
Comment (damonkelley) [inline src/main/java/clientApplication/SimpleApplication.java]: 🤔

File: src/main/java/clientApplication/SimpleApplication.java [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
+package clientApplication;
 
-import httpserver.HttpRequest;
-import httpserver.HttpResponse;
+import http.Application;
+import http.HttpRequest;
+import http.HttpResponse;
 
-public class SimpleApplication implements Application{
+public class SimpleApplication implements Application {
```
Comment (damonkelley) [inline src/main/java/clientApplication/SimpleApplication.java]: Are there any tests for the `SimpleApplication`?

Comment (Naomi-Dennis) [reply src/main/java/clientApplication/SimpleApplication.java]: Atl the tests are in spinach

File: src/main/java/clientApplication/SimpleApplication.java (hunk 2 above)
Comment (Naomi-Dennis) [reply src/main/java/clientApplication/SimpleApplication.java]: Darn. No. Super no.

**Comment by damonkelley** [comment]
@Naomi-Dennis I think you merged this into the `features/echo-body` branch instead of master.

**Pull request merged**